grumblr/app/assets/javascripts/backbone/views/formView.js

```diff
@@ -0,0 +1,28 @@
+var FormView = Backbone.View.extend({
+  el: "#grumble-form",
+
+  events:{
+    "click .create": "markCreate"
+  },
+
+  initialize: function(){
+    console.log("form has been instatiated")
+    this.render();
+  },
+
+
+  render:function(){
+    var renderedHTML = HandlebarsTemplates['grumbles/form']();
+    this.$el.html(renderedHTML);
+  },
+
+  markCreate: function(){
```

Comment:
fair enough.  I went with "markCreate" only because i had a "markDelete"
function.

On Thu, May 21, 2015 at 11:50 AM, Adam Bray <notifications@github.com>
wrote:

> In grumblr/app/assets/javascripts/backbone/views/formView.js
> <https://github.com/ga-dc/grumblr/pull/21#discussion_r30816214>:
>
> > +  events:{
> > +    "click .create": "markCreate"
> > +  },
> > +
> > +  initialize: function(){
> > +    console.log("form has been instatiated")
> > +    this.render();
> > +  },
> > +
> > +
> > +  render:function(){
> > +    var renderedHTML = HandlebarsTemplates['grumbles/form']();
> > +    this.$el.html(renderedHTML);
> > +  },
> > +
> > +  markCreate: function(){
>
> I'd prefer a clearer name for this function. Perhaps createGrumble
> instead of markCreate?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-dc/grumblr/pull/21/files#r30816214>.
>



-- 
Keith W. Knox